lib/atscppapi/examples/globalhook/GlobalHookPlugin.cc

```diff
@@ -19,24 +19,30 @@
 #include <iostream>
 #include <atscppapi/GlobalPlugin.h>
 #include <atscppapi/PluginInit.h>
-
+#include<../ts/Diags.h>
 using namespace atscppapi;
 
 class GlobalHookPlugin : public GlobalPlugin
 {
 public:
   GlobalHookPlugin() { registerHook(HOOK_READ_REQUEST_HEADERS_PRE_REMAP); }
-
+  
   virtual void
   handleReadRequestHeadersPreRemap(Transaction &transaction)
-  {
-    std::cout << "Hello from handleReadRequesHeadersPreRemap!" << std::endl;
-    transaction.resume();
+  {    
+    std::cout << "Hello from handleReadRequesHeadersPreRemap!" << std::endl;      
+    transaction.resume();       
   }
 };
 
 void
 TSPluginInit(int argc ATSCPPAPI_UNUSED, const char *argv[] ATSCPPAPI_UNUSED)
 {
+  TSPluginRegistrationInfo info;
```

Comment:
You should have a wrapper for PluginInit. It might also be reasonable to split off the fixes to the example plugins.